pkcs7/src/signer_info.rs

```diff
@@ -15,12 +15,12 @@ use x509_cert::{
 /// ```text
 /// SignedAttributes ::= SET SIZE (1..MAX) OF Attribute
 /// ```
-type SignedAttributes<'a> = SetOfVec<Attribute>;
+pub type SignedAttributes<'a> = SetOfVec<Attribute>;
```

Comment:
I believe changes in pkcs7 need to be dropped (deprecated).

Reply:
Right, I will check pkcs7 for remainders of my changes.